### ISSUE TITLE
Change text on "Auto Refresh..." button

### ISSUE THREAD

**Issue by brentfraser** [nickfox/GpsTracker#56]
"Auto refresh On" is confusing.  Does it mean Auto Refresh IS On, or click here to turn Auto Refresh On?
It should read "Auto Refresh is On" (and "Auto Refresh is Off")

**Comment by nickfox** [comment]
yes, that was a problem. The problem is that if it's longer text, then the
text is distorted. not enough room. I spent a considerable amount of time
trying to get that right and it has to be tested on all platforms (from
phone, laptop to desktop). The text needs to be short...

n
